environment/README.MD

```diff
@@ -1 +1,128 @@
-# environment
\ No newline at end of file
+# Environment
+It is a file about how to prepare environment.
+
+## Table of contents
+* [Download python](#download-python)
+* [Download pycharm](#download-pycharm)
+* [Download git](#download-git)
+* [Install Notepad++](#install-Notepad++)
+* [Create repository on github](#create-repository-on-github)
+* [Clone repository](#clone-repository-using-git-bash)
+* [Create a new branch](#create-a-new-branch)
+* [Create prepare_environment file](#create-prepare_environment-file)
+
+
+
+## Download python
+
+```
+1. Go to https://www.python.org/downloads/
```

Comment:
change this as a link https://www.python.org/downloads/

Reply:
Done